README.md

```diff
@@ -73,6 +73,8 @@ Other Style Guides
     - `symbol`
     - `bigint`
 
+    <br />
```

Comment:
i'd prefer a non-HTML way to add a line break here.

Reply:
I Just Changed the Branch and just replaced <br /> with &nbsp at ; #2467

Reply:
Please don't post duplicate PRs.